common/globalTruffleConfig.js

```diff
@@ -21,7 +21,6 @@ const infuraApiKey = process.env.INFURA_API_KEY ? process.env.INFURA_API_KEY : "
 
 // Default options
 const gasPx = 20000000000; // 20 gwei
-const gas = 9000000; // Conservative estimate of the block gas limit.
```

Comment:
could you not set `gas = undefined` here then only need to change this one line?

Reply:
Good point. Will update.

Reply:
Done.